build.sh

```diff
@@ -3,5 +3,4 @@ set -eo pipefail
 
 dotnet --info
 dotnet tool restore
-dotnet paket restore
-dotnet fake run build.fsx $@
+dotnet fsi build.fsx "$@"
```

Comment:
Note the additional quoting of the `$@` parameter, to prevent word splitting (see https://www.gnu.org/software/bash/manual/html_node/Special-Parameters.html)